java/component/keyple-calypso/src/main/java/org/eclipse/keyple/calypso/transaction/SamCommandProcessor.java

```diff
@@ -73,6 +75,7 @@
     private byte workKeyKVC;
     private boolean isDiversificationDone;
     private boolean isDigestInitDone;
+    private boolean isDigesterInitialized;
```

Comment:
isDigesterInitialized or isDigestInitialized ?

Reply:
It is really the component that calculates the digest in question and not the digest itself. Hence the name "digester". Maybe we can come up with something clearer!